React/Modules/RCTUIManager.m

```diff
@@ -354,6 +354,7 @@ - (NSString *)viewNameForReactTag:(NSNumber *)reactTag
     return name;
   }
 
+#if !TARGET_OS_OSX // [TODO(macOS GH#774)
```

Comment:
for this one can we write a short comment that we need to revisit this when we look at fabric for mac, please? thanks!